kolibri/plugins/device_management/assets/src/views/DeviceIndex.vue

```diff
@@ -18,7 +19,12 @@
     </transition>
 
     <div>
-      <component :is="currentPage" />
+      <component
+        :is="currentPage"
+        :appBarTitle.sync="childAppBarTitle"
+        :appBarLink.sync="childAppBarLink"
+        :immersivePageIcon.sync="immersivePageIcon"
```

Comment:
This feels like something that might be more conventionally done with vuex?

Curious on the thinking here to using `.sync`

Reply:
I'd rather not expand Vuex state if I can help it.

Using the `.sync` modifier is just a convenience since it avoid writing a handler like `@updateChildAppBarTitle="appBarTitle = $event"`. This shorthand could probably used in more places, even if the communication isn't always two-way.

Reply:
Yeah, my intuition is that the toolbar/corebase state would be better handled in a dedicated vuex module.

Within the code base, currently our use of 2-way communication is limited to using `v-model` in custom user input components, where there is a clear "value" associated with the component.

Using `.sync` like this seems like it saves some vuex boilerplate but breaks the spirit of one-directional data flow. I believe we should avoid letting it become a precedent.

If you do want to merge it in as-is, recommend:

1. add a comment indicating that it is a temporary fix, and should not be copy-pasted to other parts of the app
2. file a follow-up issue around migrating toolbar/corebase state to vuex

thanks!

Reply:
> This shorthand could probably used in more places, even if the communication isn't always two-way and it helps get rid of differently-named handlers.

Are you referring to `v-model` specifically? (filed #4499 - feel free to elaborate on it)